showcase/src/assets/translations/fr.json

```diff
@@ -87,6 +87,17 @@
       "LOGOUT": {
         "LOGOUT_DEFAULT_ICON": "Bouton logout - Icône par défaut",
         "LOGOUT_CUSTOM_ICON": "Bouton logout - Icône custom"
+      },
+      "COLLAPSIBLE": {
```

Comment:
I think it is good practice to order this json alphabetically :-) (even if we didn t do it 100% for now)